src/unity/toolkits/activity_classification/activity_classifier.cpp

```diff
@@ -132,9 +132,10 @@ void activity_classifier::train(gl_sframe data, std::string target_column_name,
   nn_spec_->update_params(trained_weights);
 }
 
-gl_sarray activity_classifier::predict(gl_sframe data,
-                                       std::string output_type)
-{
+
+gl_sarray activity_classifier::predict(gl_sframe data, std::string output_type,
+                                       std::string output_frequency) {
```

Comment:
Looks like this is from an earlier version of the change. We don't need `predict` to take `output_frequency` any more, right?